docker/test/develop.yml

```diff
@@ -0,0 +1,42 @@
+# This needs at least compose 1.6.0
+version: '2'
+
+services:
+  postgresdata:
+    image: postgres:9.5
+    command: /bin/true
+
+  postgres:
+    image: postgres:9.5
+    volumes_from:
+      - postgresdata
+    environment:
+      POSTGRES_PASSWORD: myhuginnpassword
+      POSTGRES_USER: huginn
+
+  huginn_test_postgres:
```

Comment:
a separate docker-compose service for for mysql (e.g. `huginn_test_mysql`) could also be added. I thought it would make sense to validate the general approach before adding this